ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml

```diff
@@ -18,6 +18,8 @@ tests:
   cron: 0 */6 * * *
   steps:
     cluster_profile: aws
+    env:
+      TEST_OPTIONS: ""
     workflow: openshift-upgrade-aws
```

Comment:
Hrm, only the Loki workflows default to `TEST_OPTIONS` setting `abort-at`.  I'm still a bit fuzzy about why this one seems to be pulling it in.

Reply:
I don't think that job is related - it appears to affect rollback jobs only